dask/array/ufunc.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import absolute_import, division, print_function
+
+from operator import getitem
+
+import numpy as np
+
+from toolz.curried import merge
+from .core import Array, elemwise
+from .. import core
+from ..utils import skip_doctest
+
+
+def wrap_elemwise(numpy_ufunc, **wrapped_kwargs):
+    """ Wrap up numpy function into dask.array """
+
+    def wrapped(x, *args, **kwargs):
+        if hasattr(x, '_elemwise_'):
+            return x._elemwise_(numpy_ufunc, x, *args, **kwargs)
+            # return x._elemwise_(numpy_ufunc, x, *args, **kwargs,
```

Comment:
Current test all succeeded even if it ignores ``wrap_elemwise`` args (like ``logical_and = wrap_elemwise(np.logical_and, dtype='bool')``). Can I remove it?